### ISSUE TITLE
''bert-large-uncased-whole-word-masking-finetuned-squad' CAN'T be reached.

### ISSUE THREAD

**Issue by sinboyxx** [huggingface/pytorch-transformers#763]
'bert-large-uncased-whole-word-masking-finetuned-squad' can't be reached from the addr in tokenization.py:

https://s3.amazonaws.com/models.huggingface.co/bert/bert-large-uncased-whole-word-masking-finetuned-squad-vocab.txt

**Comment by liuzzi** [comment]
also ran into this.  The model seems to work with the bert-large-uncased vocab, but I wonder if there are unintended consequences because of this
